### PR TITLE
Jcenter upload

### PR DESCRIPTION
**Pull request by olgalazarova**
Configure build.gradle file for uploading sdk library to maven and jcenter.

### CODE REVIEW

File: sdk/build.gradle [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+    libraryDescription = 'The official Chute SDK Android'
+
+    siteUrl = 'https://github.com/chute/Chute-SDK-Android'
+    gitUrl = 'https://github.com/chute/Chute-SDK-Android.git'
+
+    libraryVersion = '1.0.0'
+
+    developerId = 'olgalazarova'
+    developerName = 'Olga Lazarova'
+    developerEmail = 'olahla@email.com'
```
Comment (zegomesjf) [inline sdk/build.gradle]: olahla@gmail.com ?

File: sample/build.gradle [hunk 2]
```diff
@@ -20,6 +20,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    lintOptions {
+        abortOnError false
```
Comment (jorgemf) [inline sample/build.gradle]: what is the reason for this? Do we have any lint error?

File: sdk/build.gradle [hunk 3]
```diff
@@ -27,6 +53,47 @@ android {
         exclude 'META-INF/MANIFEST.MF'
         exclude 'META-INF/rxjava.properties'
     }
+    lintOptions {
+        abortOnError false
```
Comment (jorgemf) [inline sdk/build.gradle]: Do we have any lint error in the sdk? What is the reason to include this here?

File: sdk/build.gradle [hunk 4]
```diff
@@ -27,6 +53,47 @@ android {
...
+                // Set your license
+                licenses {
+                    license {
+                        name 'The Apache Software License, Version 2.0'
+                        url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                developers {
+                    developer {
+                        id 'olgalazarova'
```
Comment (jorgemf) [inline sdk/build.gradle]: you should use the variables defined before

File: sdk/build.gradle (hunk 1 above)
Comment (olgalazarova) [inline sdk/build.gradle]: I'm registered on bintray under that email

Comment (olgalazarova) [inline sdk/build.gradle]: oh, its a typo, sorry I'll correct this

File: sdk/build.gradle (hunk 3 above)
Comment (olgalazarova) [inline sdk/build.gradle]: there was one error, I just noticed it. It was the automatically generated layout that comes with the MainActivity - all fixed now

**Pull request closed without merging**